component/src/main/java/me/androidbox/component/agenda/AlarmReminder.kt

```diff
@@ -43,6 +49,14 @@ fun AlarmReminder(
         }
 }
 
+enum class AlarmReminderItem(@StringRes val text: Int) {
+    TEN_MINUTES(R.string.ten_minutes_before),
+    THIRTY_MINUTES(R.string.thirty_minutes_before),
+    ONE_HOUR(R.string.one_hour_before),
+    SIX_HOUR(R.string.six_hours_before),
+    ONE_DAY(R.string.one_day_before)
+}
```

Comment:
Just wondering, does this enum class belong in this AlarmReminder? Because I am not referencing these inside the AlarmReminder. But as its related I thought it be belong in here.

Reply:
It's okay to keep it there IMO, but could also be a separate file. But I'd rename the parameter to `stringResId` or similar, since its type `Int` doesn't really fit to the name `text` I think

Reply:
@philipplackner Renamed to stringResId